src/arch/LowLevelWindow/LowLevelWindow_SDL.cpp

```diff
@@ -0,0 +1,165 @@
+#include "global.h"
+#include "LowLevelWindow_SDL.h"
+#include "RageLog.h"
+#include "RageException.h"
+#include "archutils/Unix/X11Helper.h"
```

Comment:
RageLog is needed to print some information to the console, same goes for RageException in case initializing SDL fails.
X11Helper is needed to receive the handlers for the Display and the Window, as this is needed for handling the Input with the X11 Input driver.